src/style.css

```diff
@@ -0,0 +1,41 @@
+body {
+  margin: 0;
+}
+
+.starCard {
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
that's redundant styles, div are positioned in column by default
```suggestion
```

Reply:
Thanks

Reply:
It seems you are wrong, try it